examples/viewer.py

```diff
@@ -0,0 +1,39 @@
+from magnum import gl
+from magnum.platform.glfw import Application
+from settings import default_sim_settings, make_cfg
+
+import habitat_sim
+
+# Setting filepath to load
+sim_settings = default_sim_settings
+sim_settings[
+    "scene"
+] = "data/scene_datasets/habitat-test-scenes/frl_apartment_stage.stage_config.json"
+
+
+class SkeletonPythonViewer(Application):
+    def __init__(self):
+        configuration = self.Configuration()
+        configuration.title = "Skeleton Viewer Application"
+        Application.__init__(self, configuration)
+
+        self.viewport_size = gl.default_framebuffer.viewport.size()
+
+        self.cfg = make_cfg(sim_settings)
+        self.sim = habitat_sim.Simulator(self.cfg)
```

Comment:
Depending on some slight variations to Simulator.cpp (thanks mosra!), this line typically causes errors like
`GL::Context: Another context currently active`
or
`GL::Context: no current context`
or
`Simulator::gpuDevice: no OpenGL context.`

Reply:
@mosra ^

Reply:
@Skylion007 yeah I know, one of the persistent nightmare issues I have to look at.